enos/modules/ec2_info/main.tf

```diff
@@ -13,6 +15,13 @@ locals {
         "20.04" = data.aws_ami.ubuntu_2004["arm64"].id
         "22.04" = data.aws_ami.ubuntu_2204["arm64"].id
       }
+      "amazon_linux" = {
+        "amzn2" = data.aws_ami.amazon_linux_2["arm64"].id
```

Comment:
Should we update this to 2.0, as that's the current version? I suspect they could eventually do a 2.x before another major release, but even if they don't that would future proof it for us.